resources/publications.md

```diff
@@ -95,12 +93,57 @@ td:nth-child(2) {
 
 ## Metrics
 
+Machine translation metrics automatically assess quality of the machine translation output.
+There are two types of metrics: **quality evaluation**  and **quality estimation**.   
+* [Quality evaluation](/quality/quality-evaluation.md) metrics rely on human (reference) translation.
+* [Quality estimation](/quality/quality-estimation.md) metrics ***do not*** rely on human (reference) translation. 
+
+### Quality evaluation
+
+#### Similarity-based metrics
+
+These metrics evaluate similarity between machine translation and reference translation. This similarity can be based on:
+* n-gram matching
+* embeddings
+
+##### n-gram matching metrics
+
+These metrics evaluate similarity based on hand-crafted features and rules. For example, a metric can count the number and fraction of n-grams that appear in both the machine translation and the human translation.
+
 |     |     |     |
 | --- | --- | --- |
-| 2020 | [***COMET: A Neural Framework for MT Evaluation***](https://aclanthology.org/2020.emnlp-main.213.pdf) | Ricardo Rei, Craig Stewart, Ana C. Farinha, Alon Lavie |
+| 2015 | [***chrF: character n-gram f-score for automatic MT evaluation.***](https://www.cs.cmu.edu/~alavie/METEOR/pdf/meteor-1.5.pdf) |	Maja Popovic ́ |
```

Comment:
Seems like the Popovic accent is misplaced. From what I'm seeing at [machinetranslate.org](http://machinetranslate.org), we've spelled it without the accent. What do you think?

Reply:
Also there shouldn't be a period in the paper's name